internal/graphql/pull_request.go

```diff
@@ -222,3 +237,18 @@ func (c *Client) GetPullRequest(ctx context.Context, nodeID string) *PullRequest
 
 	return res
 }
+
+// isCurrentIteration checks if given startDate is current iteration.
+func isCurrentIteration(startDate string, duration int) bool {
```

Comment:
Yeah, let's remove that for now, and always use the value returned by GitHub API

Reply:
^

Reply:
@ronaudinho poke :)